NEWS.md

```diff
@@ -1,3 +1,10 @@
+## 14.1.2 2020-04-09
+
+* Fix processing of job profiles in json format by MatchEngine (MODINV-271)
+* Fix matching on Instance fields (MODINV-275)
+* Enable jackson serialization for EffectiveCallNumberComponent (MODINV-278)
```

Comment:
Please could this be expressed in terms of change in behaviour or impact on the user